orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/AdminController.groovy

```diff
@@ -74,6 +79,23 @@ class AdminController {
     forceExecutionCancellationCommand.forceCancel(executionType, executionId, canceledBy)
   }
 
+
+  @RequestMapping(value = "/executions/", method = RequestMethod.POST)
+  @ResponseStatus(HttpStatus.OK)
+  void createExecution(@RequestBody Execution execution) {
+
+    if (execution.status in [ExecutionStatus.CANCELED, ExecutionStatus.SUCCEEDED, ExecutionStatus.TERMINAL]) {
```

Comment:
do you think it's worth validating that the app & pipeline exist. Maybe at least spitting our an http response with some text:
```
Imported execution with ID: {} for app: {}, pipeline: {} with {n} stages
Note: the app doesn't exist in this environment
```

I think it would be valuable otherwise folks might import it and not see it and be confused?
Also, would be useful to know the ID of newly imported execution (though it's the same as the payload)